applications/MPMApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -65,11 +65,18 @@ namespace Python{
             rBackgroundGridModelPart, rInitialModelPart, rMPMModelPart);
     }
 
+    void GenerateLagrangeNodes(
+        ModelPart& rBackgroundGridModelPart)
+    {
+        MaterialPointGeneratorUtility::GenerateLagrangeNodes(rBackgroundGridModelPart);
+    }
+
     void  AddCustomUtilitiesToPython(pybind11::module& m)
     {
         m.def("SearchElement", SearchElementAccordingToDimension);
         m.def("GenerateMaterialPointElement", GenerateMaterialPointElementAccordingToDimension);
         m.def("GenerateMaterialPointCondition", GenerateMaterialPointConditionAccordingToDimension);
+        m.def("GenerateLagrangeNodes", GenerateLagrangeNodes);
```

Comment:
I would add these to make it easier in Python, which model part to pass
```suggestion
        m.def("GenerateLagrangeNodes", GenerateLagrangeNodes, pybind11::arg("BackgroundGridModelPart"));
```